website/templates/website/about.html

```diff
@@ -1,53 +1,52 @@
 {% extends 'website/base.html' %}
 {% block content %}
-<img src="/media/images/office_.jpg" style="width:100%">
-<div class="container-fluid" style="margin-top: 150px">
-    <div class="row" style="padding-left: 200px; padding-right: 200px; padding-bottom: 150px;">
+<div class="container-fluid about-container">
+<img src="/media/images/office_.jpg" class="img-top">
```

Comment:
you shouldn't have a tailing underscore in the name of the image
```suggestion
<img src="/media/images/office.jpg" class="img-top">
```